devops/actions/llvm_test_suite/action.yml

```diff
@@ -28,8 +28,12 @@ inputs:
   cmake_args:
     description: 'Extra arguments to cmake command'
     required: false
-  merge_intel_branch:
-    description: 'Whether to merge intel branch to test_ref or not'
+  target_branch:
+    description: 'Name of the target branch to be merged in PRs'
+    default: 'intel'
+    required: false
```

Comment:
Can't we use `github.base_ref` instead?

Reply:
No. `github.base_ref` is set as ref for current PR. Strictly speaking we don't always want to merge tip twice: if we test current PR with PR from another repo (like `verify_with` does), we want to merge tips into both branches, but those will be different target branches.

Reply:
How will it work if I create a PR targeting `openmp` branch then?

Reply:
These workflows only run against `sycl` branch. But you can create a workflow, that would run against `openmp` branch, and set `openmp` as target branch. The scenario I'm speaking about is different.
Imagine you have 2 PRs to intel/llvm and intel/llvm-test-suite. You need to test them together. When you write a command to run tests against PR in other repo, you would still want to merge tips of both repos into corresponding PRs before running tests. `github.base_ref` wouldn't work here, because it only contains ref for current PR, not the one in another repo.

Reply:
I understand the problem you are trying to solve, but I don't think that hardcoding branches names is a good idea.
Does `github.event.pull_request.base.ref` work?
https://github.community/t/how-to-get-the-target-branch-name-using-pull-request-review-event-on-github-actions/121875/2